scripts/portstat

```diff
@@ -388,20 +401,25 @@ Examples:
         else:
             print "Cleared counters"
             sys.exit(0)
-
-    cnstat_cached_dict = OrderedDict()
-
-    if os.path.isfile(cnstat_fqn_file):
-        try:
-            cnstat_cached_dict = pickle.load(open(cnstat_fqn_file, 'r'))
-            print "Last cached time was " + str(cnstat_cached_dict.get('time'))
-            cnstat_diff_print(cnstat_dict, cnstat_cached_dict, use_json)
-        except IOError as e:
-            print e.errno, e
-    else:
-        if tag_name:
-            print "\nFile '%s' does not exist" % cnstat_fqn_file
-            print "Did you run 'portstat -c -t %s' to record the counters via tag %s?\n" % (tag_name, tag_name)
+	
+    if wait_time_in_seconds == 0:
+        cnstat_cached_dict = OrderedDict()
+        if os.path.isfile(cnstat_fqn_file):
+            try:
+                cnstat_cached_dict = pickle.load(open(cnstat_fqn_file, 'r'))
+                print "Last cached time was " + str(cnstat_cached_dict.get('time'))
+                cnstat_diff_print(cnstat_dict, cnstat_cached_dict, use_json)
+            except IOError as e:
+                print e.errno, e
         else:
-            cnstat_print(cnstat_dict, use_json)
+            if tag_name:
+                print "\nFile '%s' does not exist" % cnstat_fqn_file
+                print "Did you run 'portstat -c -t %s' to record the counters via tag %s?\n" % (tag_name, tag_name)
+            else:
+                cnstat_print(cnstat_dict, use_json)
+    else:
+        #wait for sometime and output the result
```

Comment:
"# Wait for the specified duration then gather new stats and output the difference"